tests/inputs/example_service/test_example_service.py

```diff
@@ -0,0 +1,108 @@
+import asyncio
+from typing import AsyncIterator, AsyncIterable
+
+from grpclib.client import Channel
+from grpclib.server import Server
+
+from tests.output_betterproto.example_service.example_service import (
+    ExampleServiceImplementation,
+    ExampleServiceStub,
+    ExampleRequest,
+    ExampleResponse,
+)
+
+
+class ExampleService(ExampleServiceImplementation):
```

Comment:
This seems backwards to me. I'd usually use _Implementation_ to refer to the class that implements the abstract class for the service. 

How about: 

```suggestion
class ExampleServiceImplementation(ExampleServiceBase):
```

Reply:
I initially did this to clearly distinguish between the client-facing `ServiceStub` and the server-facing `ServiceImplementation`. But I see how this can be confusing. I renamed the ABC to `ServiceBase` and the generated class to `ExampleServiceBase`. I'd like to avoid putting "Implementation" in the example implementation name as I feel it's unnecessarily long.

How about:
```python
class ExampleService(ExampleServiceBase):
```